app/scripts/app/models/variable_exploration/variable_selection.directives.js

```diff
@@ -396,6 +396,13 @@ angular
             //   })
             //   .style("opacity", 0.2);
           };
+
```

Comment:
just an opinion but the event name could contain a verb, it would help understand what the action is.